cleanlab/outlier.py

```diff
@@ -514,6 +526,15 @@ def _get_ood_predictions_scores(
         ood_predictions_scores = 1.0 - get_normalized_entropy(pred_probs)
     elif method == "least_confidence":
         ood_predictions_scores = pred_probs.max(axis=1)
+    elif method == "gen":
+        if pred_probs.shape[1] < M:
```

Comment:
you can ask codecov to ignore this line. Defensive lines of code don't need to be unit tested